web/themes/custom/novel/templates/views/node--event-list-teaser.html.twig

```diff
@@ -0,0 +1,31 @@
+<a href="{{ content.event_link }}" class="event-list-item arrow__hover--right-small">
+  <div class="event-list-item__image-container">
+    {% if content.field_event_image['#items'] is not empty %}
+      {% set image_uri = content.field_event_image['#items'].entity.field_media_image.entity.fileuri %}
+      {% set image_url = file_url(image_uri) %}
+        <img src="{{ image_url }}" alt="{{ content.label }}" class="event-list-item__image" />
+    {% endif %}
+  </div>
+  <div class="event-list-item__content">
+    <div aria-pressed="false" class="tag tag--fill cursor-pointer tag--small event-list-item__tag">foredrag</div>
```

Comment:
Please remove `aria-pressed="false"`.

I do not think it makes sense in this context as the element is no longer a `button`.